README.md

```diff
@@ -47,7 +47,7 @@ Vagrant.configure('2') do |config|
     # provider.paymentterm = <*1*,12,24>
     # provider.datacenterid = <int>
     # provider.image = <string>
-    # provider.imageid = <int>
+    # provider.image_id = <int>
```

Comment:
Let's stick with no underscore for consistency with the other parameters.

Likewise on https://github.com/bdwyertech/vagrant-linode/blob/allow-image_id/lib/vagrant-linode/actions/create.rb#L41 (etc)